### PR TITLE
ENCD-6145-more-facet-headers

### PR DESCRIPTION
**Pull request by jenjou**

### CODE REVIEW

File: src/encoded/schemas/annotation.json [hunk 1]
```diff
@@ -271,6 +271,62 @@
             "linkTo": "Experiment"
         }
     },
+    "facet_groups": [
+        {
+            "title": "Annotation",
+            "facet_fields": [
```
Comment (gabdank) [inline src/encoded/schemas/annotation.json:277]: I think we need to include here experimental_input

Comment (gabdank) [reply src/encoded/schemas/annotation.json:277]: and biochemical inputs

File: src/encoded/schemas/biosample.json [hunk 2]
```diff
@@ -922,6 +922,66 @@
             ]
         }
     },
+    "facet_groups": [
+        {
+            "title": "Biosample",
+            "facet_fields": [
+                "biosample_ontology.classification",
+                "biosample_ontology.organ_slims",
```
Comment (gabdank) [inline src/encoded/schemas/biosample.json:930]: I think we need to include age_display

File: src/encoded/schemas/biosample.json [hunk 3]
```diff
@@ -922,6 +922,66 @@
...
+        {
+            "title": "Dataset",
+            "facet_fields": [
+                "assay_title",
+                "annotation_type",
+                "annotation_subtype",
+                "platform.term_name",
+                "target.label",
+                "targets.label",
+                "encyclopedia_version"
```
Comment (gabdank) [inline src/encoded/schemas/biosample.json]: I am confused. why is it here?

Comment (jenjou) [reply src/encoded/schemas/biosample.json]: I think your review is highlighting multiple lines, can you clarify which facet(s) you are referring to that should be moved? Is it "encyclopedia_version"?

File: src/encoded/schemas/annotation.json (hunk 1 above)
Comment (gabdank) [reply src/encoded/schemas/annotation.json:277]: do not add experimental inputs, it is not a good idea

**Review by gabdank — DISMISSED** [review]

**Comment by jenjou** [comment]
No code change, just rebased on the hotfix

**Review by gabdank — APPROVED** [review]

**Pull request merged**